extensions/amp-analytics/0.1/vendors/upscore.js

```diff
@@ -0,0 +1,74 @@
+
+
```

Comment:
Deleting these two empty lines would be fine

Reply:
No problem, I cleaned up the file, and hopefully, now the test will be triggered. Thank you @torch2424 for all the work.